pkgs/development/ocaml-modules/happy-eyeballs/mirage.nix

```diff
@@ -1,25 +1,39 @@
 { buildDunePackage
 , happy-eyeballs
+, duration
 , dns-client
+, domain-name
+, ipaddr
+, fmt
 , logs
 , lwt
+, mirage-clock
+, mirage-random
+, mirage-time
 , tcpip
 }:
 
 buildDunePackage {
   pname = "happy-eyeballs-mirage";
+
   inherit (happy-eyeballs) src version;
 
+  minimalOCamlVersion = "4.08";
+
+  strictDeps = true;
+
   propagatedBuildInputs = [
     dns-client
     happy-eyeballs
-    logs
     lwt
+    mirage-clock
+    mirage-random
+    mirage-time
     tcpip
   ];
```

Comment:
![image](https://user-images.githubusercontent.com/5861043/191566533-45972f8a-3c60-4ac0-aac0-88c7df493171.png)

`logs` is missing from propagatedBuildInputs

mirage-clock, mirage-random, mirage-time is not propagated.

Reply:
```
buildInputs = [
  mirage-clock
  mirage-random
  mirage-time
];
```

Opam package has:
* duration
* domain-name
* ipaddr
* fmt (?)

Reply:
`lwt` is already propagated by dns-client.